sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/Utxo.kt

```diff
@@ -0,0 +1,18 @@
+package cash.z.ecc.android.sdk.internal.model
+
+import cash.z.ecc.android.sdk.model.BlockHeight
+import cash.z.ecc.android.sdk.model.FirstClassByteArray
+import cash.z.ecc.android.sdk.model.Zatoshi
+
+internal data class Utxo(
+    val id: Long? = 0L,
+    val address: String,
+    val txid: FirstClassByteArray,
+    val transactionIndex: Int,
+    val script: FirstClassByteArray,
+
+    val value: Zatoshi,
+
+    val height: BlockHeight?,
+    val spent: Int? = 0
+)
```

Comment:
If this is meant to match the columns in the `utxos` table (which the existence of the `spent` field pointing to a transaction's database ID suggests), there is also a `received_by_account` column ([reference](https://github.com/zcash/librustzcash/blob/1a20fc1578baaddd18db4ac0db8aa4295c06013d/zcash_client_sqlite/src/wallet/init.rs#L422-L435)). But IDK how this was previously handled by Room.